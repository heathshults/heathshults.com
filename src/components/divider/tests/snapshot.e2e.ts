import { snapIt } from '../../../test';

const component = 'divider';

describe(component, () => {
  const snap = snapIt(component);

  describe('renders', () => {
    snap('correctly', '<hs-divider></hs-divider>');
    snap('content', '<hs-divider>Hello, world</hs-divider>');
  });
});
