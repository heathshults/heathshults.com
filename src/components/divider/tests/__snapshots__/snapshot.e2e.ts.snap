// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`divider renders content 1`] = `"<hs-divider class=\\"hydrated\\"><div class=\\"hs-divider hs-divider--solid\\"><span class=\\"hs-divider__content\\">Hello, world</span></div></hs-divider>"`;

exports[`divider renders correctly 1`] = `"<hs-divider class=\\"hydrated\\"><div class=\\"hs-divider hs-divider--solid\\"></div></hs-divider>"`;
